### PR TITLE
SUBMARINE-693. Sync the status of notebook on workbench web

### PR DESCRIPTION
**Pull request by kobe860219**
### What is this PR for?
This improvement make workbench web could sync status of notebooks. It's make UI more friendly for users.
The pr is related of [SUBMARINE-675](https://github.com/apache/submarine/commit/d104080a926d30bcc45ccd740f9489dd7365d0b5)

### What type of PR is it?
[Improvement]

### Todos
* [ ] - Task

### What is the Jira issue?
[SUBMARINE-693](https://issues.apache.org/jira/browse/SUBMARINE-693)

### How should this be tested?
[Travis CI](https://travis-ci.org/github/kobe860219/submarine/builds/751380815)

### Screenshots (if appropriate)
Sync status of notebook on web

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by kobe860219** [comment]
@lowc1012  Please help review this PR. Thanks !

File: submarine-workbench/workbench-web/src/app/services/polling.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+         }
+
+         this.scheduler = timer(0, this.interval).subscribe(() => {
+             this.iterate();
+         });
+
+         return this.poller;
+     }
+
+     private iterate() {
```
Comment (lowc1012) [inline submarine-workbench/workbench-web/src/app/services/polling.ts:72]: Please adjust indents to the same format.

File: submarine-workbench/workbench-web/src/app/services/polling.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        public stop() {
+            if (this.scheduler) {
+                this.scheduler.unsubscribe();
+            }
+        }
+
+        public getPoller() {
+            return this.poller;
+        }
+ }
```
Comment (jiwq) [inline submarine-workbench/workbench-web/src/app/services/polling.ts:105]: Need newline at end of file.

**Review by jiwq — COMMENTED** [review]
Should format the new code to keep the same code style.

**Comment by jiwq** [comment]
+1 Thanks @kobe860219 for the contribution and @lowc1012 for the review.

**Pull request closed without merging**